src/Raven.Server/Web/Authentication/AdminCertificatesHandler.cs

```diff
@@ -776,9 +776,11 @@ public async Task Edit()
                     var permissions = FormatPermissions(newCertificate);
 
                     LogAuditFor("Certificates", "CHANGE", 
-                        $"Certificate {newCertificate?.Name}. Security Clearance: {newCertificate?.SecurityClearance}. Permissions: {permissions}. TwoFactor: {string.IsNullOrEmpty(twoFactorAuthenticationKey) == false}");
```

Comment:
do we log info when 2fa properties are changed for a cert?

Reply:
only info is 2fa enabled